devdocs/settingUp.md

```diff
@@ -232,7 +232,7 @@ Client scripts are scripts that remotely manipulate data on GAE via its Remote A
 
 
 ##Troubleshooting
-Troubleshooting instructions are given [in this document](https://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded=true)
+Troubleshooting instructions are given [in this document](troubleshooting-guide.md)
```

Comment:
There are two more references to troubleshooting guide in this file (settingUp.md), fix them as well.